numba/tests/test_np_functions.py

```diff
@@ -40,7 +49,12 @@ def run_unary_real(self, pyfunc, x_types, x_values,
                  functions to be tested.
         x_types: the types of the values being tested, see numba.types
         x_values: the numerical values of the values to be tested
-        flags: flags to pass to the ComplicationnCache::ccache::compile function
+        flags: flags to pass to the ComplicationCache::ccache::compile function
```

Comment:
Yeah, I think we want that to be "CompilationCache".  The rest of the Numba source code is the "ComplicationCache".  :)